src/content/docs/en/reference/modules/astro-content.mdx

```diff
@@ -149,7 +210,35 @@ const draftBlogPosts = await getCollection('blog', ({ data }) => {
 ---
 ```
 
-[See the `Content Collection` guide](/en/guides/content-collections/#querying-collections) for example usage.
+[See the `Content Collection` guide](/en/guides/content-collections/#querying-build-time-collections) for example usage.
+
+### `getLiveCollection()`
+
+<p>
+
+**Type:** `(collection: string, filter?: (entry: CollectionEntry<collection>) => boolean) => CollectionEntry<collection>[]`
+<Since v="5.0.0" />
+</p>
+
+`getLiveCollection()` is a function that retrieves a list of live content collection entries by collection name.
+
+It returns all items in the collection by default, and accepts an optional `filter` function to narrow by entry properties. This allows you to query for only some items in a collection based on `id` or frontmatter values via the `data` object.
+
+```astro
+---
+import { getLiveCollection } from 'astro:content';
+
+// Get all `src/content/blog/` entries
+const allBlogPosts = await getLiveCollection('blog');
+
+// Only return posts with `draft: true` in the frontmatter
+const draftBlogPosts = await getLiveCollection('blog', ({ data }) => {
+  return data.draft === true;
+});
```

Comment:
```suggestion
const draftBlogPosts = await getLiveCollection('blog',  { draft: true });
```
This does need explanation though. This filter format is entirely dependent on the loader.